code/game/machinery/doors/door.dm

```diff
@@ -424,4 +425,6 @@
 /obj/machinery/door/power_change()
 	. = ..()
 	if(. && !(machine_stat & NOPOWER))
-		autoclose_in(rand(0.5 SECONDS, 3 SECONDS))

```

Comment:
we loosing random is this acceptable move?

Reply:
I think, no.

Reply:
I think so

Reply:
@optimumtact I reviewed all of your changes, everything looks fine except 427 and 428 lines. Could you explain why did you change random small-gap timer in case of occasional door's power change to that big const, please?

Reply:
no I don't think I will